po/pt_BR.po

```diff
@@ -0,0 +1,216 @@
+msgid "Use dark theme"
+msgstr "Use o tema escuro"
+
+#: src/ui/dialogs/SettingsDialog.vala:34
+msgid "Steam API keys have limited number of uses per day"
+msgstr "As chaves da API do Steam têm um número limitado de utilizações por dia"
+
+#: src/ui/dialogs/SettingsDialog.vala:34
+msgid "Generate key"
+msgstr "Gerar chave"
+
+#: src/ui/dialogs/SettingsDialog.vala:35
+msgid "Steam API key"
+msgstr "Chave da API do Steam"
+
+#: src/ui/dialogs/SettingsDialog.vala:36
+msgid "Steam installation directory"
+msgstr "Diretório de instalação do Steam"
+
+#: src/ui/dialogs/SettingsDialog.vala:41
+msgid "GOG games directory"
+msgstr "Diretório de jogos GOG"
+
+#: src/ui/dialogs/SettingsDialog.vala:47
+msgid "Humble Bundle games directory"
+msgstr "Diretório de jogos Humble Bundle"
+
+#: src/ui/dialogs/SettingsDialog.vala:61
+#: src/ui/dialogs/GameDetailsDialog.vala:30
+msgid "Close"
+msgstr "Fechar"
+
+#: src/ui/dialogs/GameInstallDialog.vala:56
+msgid "Select game installer"
+msgstr "Selecione o instalador do jogo"
+
+#: src/ui/dialogs/GameInstallDialog.vala:94
+msgid "Import"
+msgstr "Importar"
+
+#: src/ui/dialogs/GameInstallDialog.vala:97
+#: src/ui/views/GameDetailsView/GameDetailsView.vala:141
+msgid "Install"
+msgstr "Instalar"
+
+#: src/ui/views/WelcomeView.vala:33
+msgid "No enabled game sources"
+msgstr "Nenhuma fonte de jogo ativada"
+
+#: src/ui/views/WelcomeView.vala:33
+msgid "Enable some game sources in settings"
+msgstr "Ative algumas fontes de jogos nas configurações"
+
+#: src/ui/views/WelcomeView.vala:34 src/ui/views/WelcomeView.vala:59
+#: src/ui/views/GamesView/GamesView.vala:147
+msgid "Settings"
+msgstr "Configurações"
+
+#: src/ui/views/WelcomeView.vala:38
+msgid "Let's get started"
+msgstr "Vamos começar"
+
+#: src/ui/views/WelcomeView.vala:52
+msgid "Skip"
+msgstr "Pular"
+
+#: src/ui/views/WelcomeView.vala:121
+msgid "Ready"
+msgstr "Pronto"
+
+#: src/ui/views/WelcomeView.vala:127
+msgid "Authentication required"
+msgstr "Autentificação requerida"
+
+#: src/ui/views/WelcomeView.vala:132
+msgid "Authenticating..."
+msgstr "Autenticando..."
+
+#: src/ui/views/WelcomeView.vala:143
+#, c-format
+msgid "Install %s"
+msgstr "Instalar %s"
+
+#: src/ui/views/WelcomeView.vala:144
+msgid "Return to GameHub after installing"
+msgstr "Retornar ao GameHub depois de instalar"
+
+#: src/ui/views/GamesView/GamesView.vala:92
+msgid "Grid view"
+msgstr "Visualizar grade"
+
+#: src/ui/views/GamesView/GamesView.vala:93
+msgid "List view"
+msgstr "Visualizar lista"
+
+#: src/ui/views/GamesView/GamesView.vala:107
+msgid "All games"
+msgstr "Todos os jogos"
+
+#: src/ui/views/GamesView/GamesView.vala:117
+#, c-format
+msgid "%s games"
+msgstr "%s jogos"
+
+#: src/ui/views/GamesView/GamesView.vala:123
+msgid "Downloads"
+msgstr "Baixado"
+
+#: src/ui/views/GamesView/GamesView.vala:142
+msgid "Search"
+msgstr "Pesquisar"
+
+#: src/ui/views/GamesView/GamesView.vala:183
+#, c-format
+msgid "%u game"
+msgid_plural "%u games"
+msgstr[0] ""
+msgstr[1] ""
```

Comment:
@LeandroStanger And this plural (should be "one game" and "many games" if pt_BR has two plural forms)